src/components/TouristInfoCards.js

```diff
@@ -0,0 +1,46 @@
+import React, { Fragment } from "react";
+
+const TouristInfoCards = () => {
```

Comment:
I'm not sure if you talked about this in Group A, but Group B had a discussion about how it may be useful to further split up this component to show an individual card that uses props to render data.

This way, we can re-use that component all over the application, instead of always having to render multiple cards.